config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,28 @@
+# Mark existing migrations as safe
+# rubocop:disable Style/NumericLiterals
+StrongMigrations.start_after = 20221004154030
+# rubocop:enable Style/NumericLiterals
+
+# Set timeouts for migrations
+# If you use PgBouncer in transaction mode, delete these lines and set timeouts on the database user
+StrongMigrations.lock_timeout = 10.seconds
+StrongMigrations.statement_timeout = 1.hour
+
+# Analyze tables after indexes are added
+# Outdated statistics can sometimes hurt performance
+StrongMigrations.auto_analyze = true
+
+# Set the version of the production database
+# so the right checks are run in development
+StrongMigrations.target_version = 10
```

Comment:
Production (and staging) DB is actually 11 
The UAT postgres image is currently 10 (bitnami limitation we discussed the other day)
Not sure how this affects the gem? 🤷‍♂️

Reply:
Ok this should be updated to 11 then.

This configuration setting basically determines the Postgres version that `strong_migrations` checks the migrations against.

So even if your development database is version 14, it will highlight issues for the target version.